apps/condo/domains/ticket/schema/TicketComment.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Generated by `createschema ticket.TicketComment 'ticket:Relationship:Ticket:CASCADE; user:Relationship:User:CASCADE; content:Text;'`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/ticket/access/TicketComment')
+
+
+const TicketComment = new GQLListSchema('TicketComment', {
+    schemaDoc: 'Textual comment for tickets',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        ticket: {
+            schemaDoc: 'Related ticket of the comment',
+            type: Relationship,
+            ref: 'Ticket',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        // This field will take part in business logic and should be declared here explicitly.
+        // That's why `createdBy` field, generated by `tracked` plugin, is not used for association of a comment with ticket.
+        user: {
+            schemaDoc: 'User, who created the comment',
+            type: Relationship,
+            ref: 'User',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
```

Comment:
We should check that you have access to set the user to non-self value. To add TODO for that

Reply:
Or restrict to set non-self value if the user is not an admin ...

Reply:
Added in `62c88c73`